docs/cnv-configuration.md

```diff
@@ -0,0 +1,153 @@
+   temporarily doubles the amount of resources the tenant is accounted for and can impact resource quotas,
+   potentially leading to migration issues and upgrade failures.
+2. Each Virtual Machine operates within its own container. This means that the virtualization overhead, 
+   runs within each VM pod instead of centrally on the host, as in traditional virtualization. 
+   This overhead is factored into quota calculations. In traditional virtualization platforms,
+   service providers typically base VM quotas solely on RAM and CPU threads, since users don't directly request
+   resources for this overhead or expect its effects. Moreover, the resource requirements for this overhead can
+   change between versions without users being notified.
+
+As a result, deploying Virtual Machines in a multi-tenant cluster while restricting hardware consumption
+from different entities is highly challenging. It requires significant resource allocation to
+handle live migrations and manage VM infrastructure overhead. This also couples users to the
+VM implementation, making them dependent on infrastructure overhead that can change unexpectedly.
+
+### Solution: AAQ Operator
+By default, AAQ Operator allocates resources for VM-associated pods, appending a `/vmi` suffix to
+`requests/limits.cpu` and `requests/limits.memory` compute resources, derived from the VM's RAM size and CPU threads.
+
+For both limits and requests with the `/vmi` suffix, the same values will apply because, unlike pods, virtual
+machines are already limited to the virtual memory and virtual CPU that were allocated to them.
+
+### Application Aware Resource Quota
+The ApplicationAwareResourceQuota API is compatible with the native ResourceQuota, meaning it shares the same specification and status definitions.
+
+Example manifest:
+
+```yaml
+apiVersion: aaq.kubevirt.io/v1alpha1
+kind: ApplicationAwareResourceQuota
+metadata:
+  name: example-resource-quota
+spec:
+  hard:
+    requests.memory: 1Gi
+    limits.memory: 1Gi
+    requests.cpu/vmi: "1"
+    requests.memory/vmi: 1Gi
+```
+
+### Application Aware Cluster Resource Quota
+This quota is API compatible with ClusterResourceQuota, meaning it shares the same specification and status definitions.
+
+The main differences between the AAQ cluster quota and AAQ quota are:
+
+- unlike AAQ quotas AAQ cluster quotas are not namespaced.
+- When creating AAQ cluster quota, you can select multiple namespaces based
+  on annotation selection, label selection, or both though`spec.selector.labels` or
+  `spec.selector.annotations`. These fields follows the standard selector format.
+
+
+  **Note**: If both `spec.selector.labels` and`spec.selector.annotations` are set only namespaces
+  that match both will be selected
+
+  **Note**: AAQ must be configured to allow AAQ cluster quota management
+
+  Refer to the "Configure AAQ Operator" section for more details. 
```

Comment:
I think its better to embed a link to the section

Reply:
Done